infra/main.bicepparam

```diff
@@ -1,8 +1,8 @@
 using 'main.bicep'
 
-param environmentName = readEnvironmentVariable('AZURE_ENV_NAME', 'env_name')
-param location = readEnvironmentVariable('AZURE_LOCATION', 'location')
-param openAiHost = readEnvironmentVariable('OPENAI_HOST', 'azure')
-param openAiApiKey = readEnvironmentVariable('OPENAI_API_KEY', 'openai_api_key')
-param openAiModel = readEnvironmentVariable('OPENAI_MODEL', 'gpt-35-turbo')
-param openAiEmbeddingModel = readEnvironmentVariable('OPENAI_EMBEDDING_MODEL', 'text-embedding-ada-002')
+param environmentName = readEnvironmentVariable('AZURE_ENV_NAME', '')
+param location = readEnvironmentVariable('AZURE_LOCATION', '')
+param openAiHost = readEnvironmentVariable('OPENAI_HOST', '')
+param openAiApiKey = readEnvironmentVariable('OPENAI_API_KEY', '')
+param openAiModel = readEnvironmentVariable('OPENAI_MODEL', '')
+param openAiEmbeddingModel = readEnvironmentVariable('OPENAI_EMBEDDING_MODEL', '')
```

Comment:
I don't mind having the models defaulted, I just didn't want the other ones defaulted.

Reply:
@jongio We have updated by your comments, please re-review.